packages/contracts/contracts/optimistic-ethereum/OVM/predeploys/OVM_SequencerFeeVault.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0 <0.8.0;
+
+/* Library Imports */
+import { Lib_PredeployAddresses } from "../../libraries/constants/Lib_PredeployAddresses.sol";
+
+/* Contract Imports */
+import { OVM_ETH } from "../predeploys/OVM_ETH.sol";
+
+/**
+ * @title OVM_SequencerFeeVault
+ * @dev Simple holding contract for fees paid to the Sequencer. Likely to be replaced in the future
+ * but "good enough for now".
+ *
+ * Compiler used: optimistic-solc
+ * Runtime target: OVM
+ */
+contract OVM_SequencerFeeVault {
+
+    /*************
+     * Constants *
+     *************/
+
+    // Minimum ETH balance that can be withdrawn in a single withdrawal.
+    uint256 public constant MIN_WITHDRAWAL_AMOUNT = 10 ether;
```

Comment:
We need to decide what this minimum is going to be. Alternatively, I could also make this a config variable in geth like l1 fee wallet.

Reply:
10 felt a little high to me but I'm not super opinionated, as long as we have test coverage

Reply:
We settled on 15 ETH minimum (discussion happened out of band).